### PR TITLE
Update privacy policy to include Companion Apps.

### PR DESCRIPTION
**Pull request by JBassett**
## Proposed change
Google Play Store has requested we mention what is done with location data that the application is collecting.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
The current Android app is being held in review due to the privacy policy not mentioning what is done with the location data it collects.
https://github.com/home-assistant/android/pull/1075

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

**Comment by balloob** [comment]
Can you update the change log at the bottom.

**Comment by JBassett** [comment]
@balloob  updated the change log

**Review by balloob — APPROVED** [review]
Love how it's exactly 3 years 👍

File: source/privacy/index.markdown [hunk 1]
```diff
@@ -46,13 +46,17 @@ If Home Assistant, or substantially all of its assets, were acquired, or in the
 
 Ads appearing on any of our websites may be delivered to users by advertising partners, who may set cookies. These cookies allow the ad server to recognize your computer each time they send you an online advertisement to compile information about you or others who use your computer. This information allows ad networks to, among other things, deliver targeted advertisements that they believe will be of most interest to you. This Privacy Policy covers the use of cookies by Home Assistant and does not cover the use of cookies by any advertisers.
 
+## Companion Applications
+All information collected by the applications are sent directly to your hosted instance of Home Assistant.  This includes, but is not limited to, sensor data and location data, which at times is sent while the application is in the background.  At no point is this data logged or given to a 3rd party.
```
Comment (frenck) [inline source/privacy/index.markdown:50]: Missing space after the header, `are` -> `is`, and no double spaces.

Cloud you open a PR to correct that @JBassett?  Thanks 👍 

```
## Companion Applications

All information collected by the applications is sent directly to your hosted instance of Home Assistant. This includes, but is not limited to, sensor data and location data, which at times is sent while the application is in the background. At no point is this data logged or given to a 3rd party.
```

**Pull request merged**